sdk/communication/communication-call-automation/src/callMedia.ts

```diff
@@ -185,7 +185,7 @@ export class CallMedia {
   /**
    * Start continuous Dtmf recognition by subscribing to tones.
    * @param targetParticipant - Target participant.
```

Comment:
Is options param mandatory for start/stop continuous methods?

Reply:
Same for send dtmf. For .net and java we decided to remove options param to keep the apis simple.

Reply:
No, it's optional object to pass all optional parameters.